.github/workflows/go.yml

```
@@ -0,0 +1,54 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+
+    - name: Build main
+      run: go build main.go
+
+    - name: Run Go Tests
```

Tests are currently broker and I think it's better to disable that action until they are not fixed in https://github.com/devfile/parser/issues/18
```suggestion
      # tests are disabled since the are broken
      # will be fixed in https://github.com/devfile/parser/issues/18
    - if: false
      name: Run Go Tests
```

---

Agreed

---

commented test target in [9fb6014](https://github.com/devfile/parser/pull/20/commits/9fb6014609641dc7e3303be4795b04b3ece55289)